config/jobs/kubernetes-sigs/sig-windows/sig-windows-sac.yaml

```diff
@@ -0,0 +1,386 @@
+- interval: 24h
+  name: ci-kubernetes-e2e-aks-engine-azure-master-windows-2004-containerd
+  decorate: true
+  decoration_config:
+    timeout: 3h
+  labels:
+    preset-service-account: "true"
+    preset-azure-windows: "true"
+    preset-windows-repo-list-2004: "true"
+    preset-azure-cred: "true"
+    preset-k8s-ssh: "true"
+    preset-dind-enabled: "true"
+    preset-windows-private-registry-cred: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200802-ff98b58-master
+      command:
+      - runner.sh
+      - kubetest
+      args:
+      # Generic e2e test args
+      - --test
+      - --up
+      - --down
+      - --build=quick
+      - --dump=$(ARTIFACTS)
+      # Azure-specific test args
+      - --deployment=aksengine
+      - --provider=skeleton
+      - --aksengine-admin-username=azureuser
+      - --aksengine-admin-password=AdminPassw0rd
+      - --aksengine-creds=$(AZURE_CREDENTIALS)
+      - --aksengine-download-url=https://aka.ms/aks-engine/aks-engine-k8s-e2e.tar.gz
+      - --aksengine-public-key=$(K8S_SSH_PUBLIC_KEY_PATH)
+      - --aksengine-private-key=$(K8S_SSH_PRIVATE_KEY_PATH)
+      - --aksengine-winZipBuildScript=$(WIN_BUILD)
+      - --aksengine-orchestratorRelease=1.18
+      - --aksengine-template-url=https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/job-templates/kubernetes_2004_containerd.json
+      - --aksengine-win-binaries
+      - --aksengine-deploy-custom-k8s
+      - --aksengine-agentpoolcount=2
+      # Specific test args
+      - --test_args=--ginkgo.flakeAttempts=2 --node-os-distro=windows --docker-config-file=$(DOCKER_CONFIG_FILE) --ginkgo.focus=\[Conformance\]|\[NodeConformance\]|\[sig-windows\]|\[sig-apps\].CronJob|\[sig-api-machinery\].ResourceQuota|\[sig-scheduling\].SchedulerPreemption|\[sig-autoscaling\].\[Feature:HPA\]  --ginkgo.skip=\[LinuxOnly\]|\[Serial\]|GMSA|Guestbook.application.should.create.and.stop.a.working.application
+      - --ginkgo-parallel=6
+      securityContext:
+        privileged: true
+  annotations:
+    testgrid-dashboards: sig-windows-sac, sig-windows-containerd, sig-windows-azure, provider-azure-periodic, provider-azure-windows
+    testgrid-tab-name: aks-engine-azure-windows-2004-master-containerd
+    testgrid-alert-email: kubernetes-provider-azure@googlegroups.com
+    description: Runs SIG-Windows release tests on K8s clusters provided by aks-engine (https://github.com/Azure/aks-engine) on Azure cloud
+- interval: 48h
+  name: ci-kubernetes-e2e-aks-engine-azure-master-windows-2004-containerd-serial-slow
+  decorate: true
+  decoration_config:
+    timeout: 5h
+  labels:
+    preset-service-account: "true"
+    preset-azure-cred: "true"
+    preset-azure-windows: "true"
+    preset-windows-repo-list: "true"
```

Comment:
This should be ``preset-windows-repo-list-2004``, so it will get the ``image-repo-list`` file for 20.04 tests. With that file, for the pulling from private registry test, it will pull from ``e2eprivate`` registry, which contains the 20.04 image in its manifest list.